main.py

```diff
@@ -86,20 +103,54 @@ def set_up_logging():
 def get_args():
     global args
     if args is None:
-        parser = argparse.ArgumentParser()
+        # Create a parser with the log level argument
+        parser = argparse.ArgumentParser(add_help=False)  # Don't add help to avoid conflict with main parser
         parser.add_argument(
-            "-l",
             "--log",
```

Comment:
Need to confirm that `-l` still works, else the README has to be updated to reflect the use of `--log` instead of `-l`.